pkg/fileutil/fileutil.go

```diff
@@ -238,3 +238,13 @@ func IsExecutable(path string) bool {
 	}
 	return false
 }
+
+// IsDeviceNode checks if the given path points to a block or char device.
+func IsDeviceNode(path string) bool {
```

Comment:
This should mention somewhere (docstring?) that it doesn't follow symlinks on purpose.